taxi/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class LicenseValidationMixin:
+    def clean_license_number(self) -> ValidationError | str:
+        license_number = self.cleaned_data["license_number"]
+        if (len(license_number) != 8
+                or not license_number[:3].isalpha()
+                or license_number[:3] != license_number[:3].upper()
+                or not license_number[3:].isnumeric()):
+            raise ValidationError(
+                "Driver license must be only 8 characters length, "
+                "first 3 uppercase letters "
+                "and last 5 characters must be digits. ")
+        return license_number
```

Comment:
It appears too complex. For a shorter and more readable approach, consider using a regular expression pattern for validating the license number.